extensions/markdown-language-features/notebook/index.ts

```diff
@@ -7,6 +7,9 @@ import * as DOMPurify from 'dompurify';
 import MarkdownIt from 'markdown-it';
 import type * as MarkdownItToken from 'markdown-it/lib/token';
 import type { ActivationFunction } from 'vscode-notebook-renderer';
+import { editorErrorForeground, editorWarningForeground, editorInfoForeground,
```

Comment:
This is an extension so you can't import from `vs` 

All of these colors are instead already exposed as css variables

Reply:
I thought something like that might be the case. How can I find out what the CSS variable names are?